### PR TITLE
Changes for aucore-blockvote-5 - remove Must Support

### PR DESCRIPTION
**Pull request by dt-r**
This PR addresses issues from the AU Core R1 Ballot for Comment aucore-blockvote-5: https://confluence.hl7.org/display/HAFWG/aucore-blockvote-5

- Removed Must Support from Location.description in AU Core Location [FHIR-45029](https://jira.hl7.org/browse/FHIR-45029).
- Removed Must Support from Organization.alias in AU Core Organization [FHIR-44585](https://jira.hl7.org/browse/FHIR-44585).
- Removed Must Support from the following elements in AU Core Patient:
  - Patient.extension:birthPlace [FHIR-45018](https://jira.hl7.org/browse/FHIR-45018)
  - Patient.extension:interpreterRequired [FHIR-45186](https://jira.hl7.org/browse/FHIR-45186)
  - Patient.birthDate.extension:birthTime [FHIR-45028](https://jira.hl7.org/browse/FHIR-45028)
  - Patient.birthDate.extension:accuracyIndicator [FHIR-45028](https://jira.hl7.org/browse/FHIR-45028)
- Removed Must Support from the following elements in AU Core Practitioner:
  - Practitioner.communication [FHIR-44588](https://jira.hl7.org/browse/FHIR-44588)
  - Practitioner.qualification, Practitioner.qualification.identifier, Practitioner.qualification.code, Practitioner.qualification.period, Practitioner.qualification.issuer: [FHIR-45118](https://jira.hl7.org/browse/FHIR-45118), [FHIR-44587](https://jira.hl7.org/browse/FHIR-44587), [FHIR-44588](https://jira.hl7.org/browse/FHIR-44588)
- Removed Must Support from PractitionerRole.location in AU Core PractitionerRole [FHIR-43841](https://jira.hl7.org/browse/FHIR-43841).
- Updated change log
- Updated examples to remove Must Support elements, noting some instances of PractitionerRole.location were retained in support of the AU Core Location profile.

### CODE REVIEW

File: input/pagecontent/changes.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ This change log documents the significant updates and resolutions implemented fr
   - Patient.extension:interpreterRequired [FHIR-45186](https://jira.hl7.org/browse/FHIR-45186)
   - Patient.birthDate.extension:birthTime [FHIR-45028](https://jira.hl7.org/browse/FHIR-45028)
   - Patient.birthDate.extension:accuracyIndicator [FHIR-45028](https://jira.hl7.org/browse/FHIR-45028)
+- Removed Must Support from the following elements in AU Core Practitioner:
+  - Practitioner.communication [FHIR-45018](https://jira.hl7.org/browse/FHIR-45018)
+  - Practitioner.qualification, Practitioner.qualification.identifier, Practitioner.qualification.code, Practitioner.qualification.period, Practitioner.qualification.issuer: [FHIR-45118](https://jira.hl7.org/browse/FHIR-45118), [FHIR-44587](https://jira.hl7.org/browse/FHIR-44587), [FHIR-44588](https://jira.hl7.org/browse/FHIR-44588)
```
Comment (dbojicic) [inline input/pagecontent/changes.md:19]: Remove mention of FHIR-44588 as it's about the removal of MS from Practitioner.communication

**Review by dbojicic — COMMENTED** [review]
One minor comment to address, pending another PR to remove the STU notes.

File: input/pagecontent/changes.md (hunk 1 above)
Comment (dt-r) [reply input/pagecontent/changes.md:19]: Fixed.

**Comment by dt-r** [comment]
> One minor comment to address, pending another PR to remove the STU notes.

Master is merged. That removes the STU notes.

**Pull request merged**